public/index.html

```diff
@@ -47,14 +47,32 @@
   <link rel="icon" type="image/png" sizes="96x96" href="/img/icons/favicon-96x96.png">
   <link rel="icon" type="image/png" sizes="16x16" href="/img/icons/favicon-16x16.png">
   <link rel="icon" type="image/png" href="img/icons/favicon-96x96.png" />
+
+  <style>
+    .preloader {
+      height: 100vh;
+      width: 100vw;
+      background: white;
+      position: fixed;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      top: 0;
+      left: 0;
+    }
+  </style>
 </head>
 
 <body style="background: #15182d">
   <noscript>
     <strong>We're sorry but Lunie doesn't work properly without JavaScript enabled. Please enable it to
       continue.</strong>
   </noscript>
-  <div id="app"></div>
+  <div id="app">
+    <div class="preloader">
```

Comment:
vue overwrites this content when it is initialized

Reply:
cool!